homeassistant/components/kostal_plenticore/switch.py

```diff
@@ -164,8 +164,3 @@ def is_on(self) -> bool:
                 f"{self.platform_name} {self._name} {self.off_label}"
             )
         return bool(self.coordinator.data[self.module_id][self.data_id] == self._is_on)
-
-    @property
-    def extra_state_attributes(self) -> dict[str, Any]:
-        """Return the state attributes."""
-        return {"last_run_success": self._last_run_success}
```

Comment:
Can we remove the `self._last_run_success` attribute now?

Reply:
Yes, I first kept it when starting with switch.py, because it's part of sensor.py, but I never added it to select.py, because I did not see the advantage of including it.

Reply:
We can remove the instance attribute itself too, right?

Reply:
yes, fixed.